app/components/view-section-table-row/macro.njk

```diff
@@ -12,9 +12,11 @@
     {% if layout == 'horizontal' %}
       <div {{ commonAttributes | safe }}>
         <div data-test-id="view-section-table-row-horizontal" class="nhsuk-grid-row">
-          <div data-test-id="view-section-table-row-title" class="nhsuk-u-font-weight-bold nhsuk-grid-column-one-half">
-            {{ params.questionTitle }}
-          </div>
+          {% if params.questionTitle %}
+            <div data-test-id="view-section-table-row-title" class="nhsuk-u-font-weight-bold nhsuk-grid-column-one-half">
+              {{ params.questionTitle }}
+            </div>
+          {% endif %}
```

Comment:
Can you add unit tests for this please. Test that it renders with and without the title for this component.

Reply:
done